src/past/translation/__init__.py

```diff
@@ -395,7 +396,7 @@ def load_module(self, fullname):
 
                         if detect_python2(source, self.pathname):
                             source = self.transform(source)
-                            with open('/tmp/futurized_code.py', 'w') as f:
+                            with open(os.path.join(tempfile.mkdtemp(),'futurized_code.py'), 'w') as f:
```

Comment:
This doesnt actually work. The existing code assumes that the futurized_code.py is in the same path as above.

Reply:
+1 -- it might make sense to call tempfile.mkdtemp() once during test setup, refer to it throughout, and delete on cleanup.

Reply:
See #335